forem/forem
# Admin comments page with flag and quality reaction details.

## What type of PR is this? (check all applicable)


## Description

Add flags & quality reactions table for each comment on admin page.

[Figma](https://www.figma.com/file/yhg9HTZDFIeWuvBt9ygrss/Moderation?type=design&node-id=1649-2637&t=PXCdFbwesYZDQwhJ-4) (see Comment page section)

## Related Tickets & Documents



- Closes #19491

## QA Instructions, Screenshots, Recordings

**Demo Video**: https://www.loom.com/share/de8f0f4439f046e689d1ad6dffbc43af?sid=54a2265b-88e1-4b5b-a1cd-75f52b92a25f

**How to test?**
1. Visit `admin/content_manager/comments/` as admin
2. Click on `View Details` button of any comment
3. Check the bottom table
4. Mark the comment as flagged or add a quality reaction by visit the mod section of the comment. (Shown in above video).
5. See the changes in table on the admin page.

**Output**

<img width="965" alt="Screenshot 2023-06-30 at 11 11 04 AM" src="https://github.com/forem/forem/assets/9396084/0113a3a8-6a1c-41f8-869e-de740caa3f40">
<img width="743" alt="Screenshot 2023-06-30 at 11 11 45 AM" src="https://github.com/forem/forem/assets/9396084/85246639-43a8-4e72-b5c1-9ea2a288d0a2">
<img width="752" alt="Screenshot 2023-06-30 at 11 11 35 AM" src="https://github.com/forem/forem/assets/9396084/0a3eb59e-0027-48d0-b6aa-45f77f0830c1">
<img width="753" alt="Screenshot 2023-06-30 at 11 11 19 AM" src="https://github.com/forem/forem/assets/9396084/878e87a0-8330-4889-b8dc-2f731aae8598">
<img width="744" alt="Screenshot 2023-06-30 at 11 11 14 AM" src="https://github.com/forem/forem/assets/9396084/bb49993c-54e5-47ea-81b8-051969937b2f">


### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?
